packages/adapters/src/process-instance-adapter.ts

```diff
@@ -76,7 +76,9 @@ IComponent {
     getPythonpath() {
         let pythonpath = path.resolve(
             __dirname,
-            isTSNode ? "../../../python_modules" : "../../python_modules"
+            isTSNode
+                ? "../../python-runner/dist"
+                : path.resolve(__dirname, require.resolve("@scramjet/python-runner"), "..")
```

Comment:
```suggestion
           path.resolve(__dirname, require.resolve("@scramjet/python-runner"), "..")
```

Reply:
similarly to the above, I want to be able to run from where the source code is without building.

Reply:
ok, so why there is `dist` folder involved?